tools/stern/src/Stern/Intra.hs

```diff
@@ -744,12 +746,12 @@ getUserConversations uid = do
             b
             ( method GET
                 . header "Z-User" (toByteString' uid)
-                . path "/conversations"
+                . path "/v2/conversations"
                 . queryItem "size" (toByteString' batchSize)
                 . maybe id (queryItem "start" . toByteString') start
                 . expect2xx
             )
-      parseResponse (mkError status502 "bad-upstream") r
+      unVersioned @'V2 <$> parseResponse (mkError status502 "bad-upstream") r
```

Comment:
this wouldn't have happened if we used servant-client for calling internal api end-points...  :)